src/PrestaShopBundle/Controller/Admin/Sell/Catalog/HorizontalFeatureController.php

```diff
@@ -0,0 +1,187 @@
+<?php
```

Comment:
Should'nt this controller be part of some documentation instead of adding it to the core ? :/

Reply:
IMO it should not exist at all, but @jolelievre insists to keep a working example temporary until we are sure which parts of horizontal migration can still be used and which can be removed.

Reply:
Indeed I requested it, a lot of work has been done during horizontal migration to be independent from the `AdminLegacyLayoutController` which is used in all migrated pages, this is the only page that we have to use as an example and I think it'll be useful when we tackle the legacyLayout migration

So instead of going back into git history or copy the code later I prefer it to be still available somewhere, the legacy layout will ba hanled in v9.0 so once it's done this will be removed anyway

Reply:
can we comment it so we can be sure this has no impact in any form ?